projects/igniteui-angular/src/lib/combo/combo.component.spec.ts

```diff
@@ -2053,266 +912,101 @@ describe('igxCombo', () => {
-                .nativeElement as HTMLElement;
-            const hasScrollbar = scrollbarContainer.scrollHeight > scrollbarContainer.clientHeight;
-            expect(hasScrollbar).toBeFalsy();
-        }));
-        it('Should display vertical scrollbar when items do not fit into the container', fakeAsync(() => {
-            const fixture = TestBed.createComponent(IgxComboTestComponent);
-            fixture.detectChanges();
-            const dropdown = fixture.componentInstance.combo.dropdown;
-            dropdown.toggle();
-            tick();
-            fixture.detectChanges();
-            const scrollbarContainer = fixture.debugElement
-                .query(By.css('.' + CSS_CLASS_SCROLLBAR_VERTICAL))
-                .nativeElement as HTMLElement;
-            const hasScrollbar = scrollbarContainer.scrollHeight > scrollbarContainer.clientHeight;
-            expect(hasScrollbar).toBeTruthy();
-        }));
     });
-
     describe('Binding tests: ', () => {
-        it('Should bind combo data to array of primitive data', () => {
-            const fixture = TestBed.createComponent(IgxComboTestComponent);
+        configureTestSuite();
+        beforeAll(async(() => {
+            TestBed.configureTestingModule({
+                declarations: [
+                    IgxComboSampleComponent,
+                    IgxComboInContainerTestComponent,
+                    IgxComboRemoteDataComponent,
+                    ComboModelBindingComponent
+                ],
+                imports: [
+                    IgxComboModule,
+                    NoopAnimationsModule,
+                    IgxToggleModule,
+                    ReactiveFormsModule,
+                    FormsModule
+                ]
+            }).compileComponents();
+        }));
+        it('should bind combo data to array of primitive data', () => {
+            fixture = TestBed.createComponent(IgxComboInContainerTestComponent);
             fixture.detectChanges();
             const data = [...fixture.componentInstance.citiesData];
-            const combo = fixture.componentInstance.combo;
+            combo = fixture.componentInstance.combo;
             const comboData = combo.data;
             expect(comboData).toEqual(data);
         });
-        it('Should bind combo data to array of objects', () => {
-            const fixture = TestBed.createComponent(IgxComboSampleComponent);
+        it('should bind combo data to array of objects', () => {
+            fixture = TestBed.createComponent(IgxComboSampleComponent);
             fixture.detectChanges();
             const data = [...fixture.componentInstance.items];
-            const combo = fixture.componentInstance.combo;
+            combo = fixture.componentInstance.combo;
             const comboData = combo.data;
             expect(comboData).toEqual(data);
         });
-        it('Should bind combo data to remote service data', (async () => {
```

Comment:
It will be better to leave the test as async, it will be more readable and also you can try to mock the NgZone  as it is done in Multi-row Layout